plugin.js

```diff
@@ -682,189 +496,423 @@
 		};
 	}
 
-	placeholdersCollection = new PlaceholdersCollection();
+	/**
+	 * @class
+	 * @constructor
+	 * @param {CKEDITOR.editor} editor
+	 */
+	function PlaceholderElementsPlugin(editor)
+	{
+		var _this = this;
```

Comment:
usually with objects I use this javascript idiom:
```javascript
var self = this;
```
and then use `self` exclusively. I had seen `_this` elsewhere and assumed you used a similar idiom.